scripts/create-plugin.js

```diff
@@ -24,7 +24,7 @@ console.log('🔣 adding metadata to package.json')
 const pkg = JSON.parse(fs.readFileSync('package.json'))
 
 pkg.main = 'lib'
-pkg.version = '0.0.0-development'
+pkg.version = '0.1.0'
```

Comment:
**question**: is the intention here for an in-development plugin to be created at 0.1, then release-please will bump that to 1.0? or will it do that without a breaking change?

Reply:
sounds good. it's a little counterintuitive to me that it needs a "breaking" change to go to 1.0 but i guess that's just a convention, and it won't happen often?

Reply:
I'd imagine in most cases we'd want to do a commit with a `release-as` footer when going to a 1.0 just so it's a bit clearer that it's going stable. To be honest, the alternative behaviour where breaking changes cause a minor bump for premajor versions just seems more intuitive to me, but it might be surprising for others.

Reply:
i think we should do minor releases for 0.x breaking changes, and i like the `release-as` for 1.0 (as long as that's documented/well-known!)

Reply:
cool, I'll update the config and documentation then!